app/views/microposts/_micropost.html.erb

```diff
@@ -0,0 +1,14 @@
+<li id="micropost-<%= micropost.id %>">
+  <%= link_to gravatar_for(micropost.user, size: Settings.gravatar_size), micropost.user %>
+  <span class="user"><%= link_to micropost.user.name, micropost.user %></span>
+  <span class="content">
+    <%= micropost.content %>
+    <%= image_tag micropost.picture.url if micropost.picture? %>
+  </span>
+  <span class="timestamp">
+    <%= t ".posted_time", time: time_ago_in_words(micropost.created_at) %>
```

Comment:
You should reseach `format date in I18n` then edit this code time `<%= t ".posted_time", time: time_ago_in_words(micropost.created_at) %>`